ladder/src/EventPoller.cpp

```diff
@@ -222,9 +222,26 @@ void EventPoller::SetWakeupCallback(const std::function<void()>& callback) {
 
 #ifdef LADDER_OS_UNIX
 Pipe::Pipe() {
+#ifndef LADDER_OS_MAC
   if (::pipe2(fd_, O_NONBLOCK) < 0) {
     EXIT("pipe2");
   }
+        const auto r = ::pipe2(&pipe_[0], O_CLOEXEC);
+        if (-1 == r)
+            throw exception("pipe2 failed: " +
```

Comment:
Please do not use `std::exception`. Instead use `EXIT` from `ladder/include/utils.h` to keep consistent with the repo